woof-code/3builddistro

```diff
@@ -332,6 +332,9 @@ if [ -n "$PETBUILDS" ]; then
 		echo "Building without qemu-user-static"
 	fi
 
+	# set EXTRA_PROCESSING in _00build.conf to perform any processing required - e.g. for devx
```

Comment:
Why is this manipulation of devx done only if PETBUILDS is non-empty? I'd do it earlier, at line 306, https://github.com/puppylinux-woof-CE/woof-CE/blob/d5b9484598f2cfa28328ce841eb4d8be6e1d43c2/woof-code/3builddistro#L306